src/components/Label/Label.module.scss

```diff
@@ -29,77 +33,115 @@
   transform-origin: bottom left;
 }
 
-.kindFill.colorPrimary {
-  background-color: var(--primary-color);
-}
+.kindFill {
+  &.colorPrimary {
+    background-color: var(--primary-color);
 
-.kindFill.colorPrimary svg path {
-  fill: var(--primary-color);
-}
+    svg path {
+      fill: var(--primary-color);
+    }
 
-.kindFill.colorDark {
-  background-color: var(--inverted-color-background);
-}
+    &.clickable:hover,
+    &.clickable:active {
+      background-color: var(--primary-hover-color);
+    }
+  }
 
-.kindFill.colorDark svg path {
-  fill: var(--inverted-color-background);
-}
+  &.colorDark {
+    background-color: var(--inverted-color-background);
```

Comment:
```
&.clickable:hover,
&.clickable:active
```
are missing here on purpose?

Reply:
Yes, they have the same colors whatsoever